hack/build-cross.sh

```diff
@@ -22,7 +22,7 @@ set -o pipefail
 
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 
-echo "NOTE: $0 has been replaced by 'make cross'"
+echo "NOTE: $0 has been replaced by 'make cross'. Run 'make help' for help."
```

Comment:
That's exactly why these scripts haven't been removed.. there are possibly other scripts calling these, so that deprecation message is to let them know to change the workflow. If there isn't something to be gained by breaking those workflows, then we shouldn't.